ps5Controller/ps5Controller/HomeModule/HomeViewController.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+
+protocol HomeViewInterface {
+    func configureInitialView()
+    func reloadData()
+    func animateWhenReloadData()
+}
+
+final class HomeViewController: UIViewController {
+    @IBOutlet weak private var collectionView: UICollectionView!
+    @IBOutlet var tabButtons: [UIButton]!
+    var presenter: HomePresenter?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter?.notifyViewLoaded()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        presenter?.notifyViewDidAppear()
+    }
+    
+    @IBAction func tabButtonsTapped(_ sender: UIButton) {
+        guard let index = tabButtons.firstIndex(of: sender),
+              let type = HomeTabType(rawValue: index) else { return }
+        tabButtons.forEach { $0.setBackgroundImage(UIImage(named: "unselectedHomeTab"), for: .normal)}
+        tabButtons.forEach { $0.tintColor = UIColor(named: "unselectedTabTintColor")}
+        tabButtons[index].setBackgroundImage(UIImage(named: "selectedHomeTab"), for: .normal)
+        tabButtons[index].tintColor = UIColor(named: "selectedTabTintColor")
+        presenter?.changeTab(type: type)
+    }
+}
+
+extension HomeViewController: HomeViewInterface {
+    func configureInitialView() {
+        navigationController?.isNavigationBarHidden = true
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        collectionView.register(HomeCollectionViewCell.nib(),
+                                forCellWithReuseIdentifier: HomeCollectionViewCell.identifier)
+    }
+    
+    func reloadData() {
+        DispatchQueue.main.async {
+            self.collectionView.reloadData()
+        }
+    }
+    
+    func animateWhenReloadData() {
+        self.collectionView.transform = CGAffineTransform(translationX: view.frame.width, y: 0)
+        UIView.animate(withDuration: 1) {
+            self.collectionView.transform = CGAffineTransform(translationX: 0, y: 0)
+        }
+    }
+}
+
+extension HomeViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        guard let homeViewModels = presenter?.getHomeViewModels() else { return 0 }
+        return homeViewModels.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: HomeCollectionViewCell.identifier, for: indexPath) as? HomeCollectionViewCell,
+              let homeViewModels = presenter?.getHomeViewModels() else { return UICollectionViewCell() }
```

Comment:
`presenter.cellForItem` olmalı, cell'i force'lamak best practice miş. Onu yeni öğrendim :D  
`let cell = collectionView.dequeueReusableCell(withReuseIdentifier: HomeCollectionViewCell.identifier, for: indexPath) as! HomeCollectionViewCell`
şeklinde olabilir